charts/vald/templates/rebalancer/jobconfigmap.yaml

```diff
@@ -26,7 +26,7 @@ metadata:
     app.kubernetes.io/version: {{ .Chart.Version }}
     app.kubernetes.io/component: rebalancer
 data:
-  {{ .Values.rebalancer.rebalancer.rebalance_job_template_key }}: |
+  job.tpl: |
```

Comment:
should we define a key in charts to store this file name?

Reply:
Since the contents of the job template are set via values.yml, I thought it would be okay if the key name was static as well as config.yml. 
https://github.com/vdaas/vald/blob/1be6a01131f9e1464c2b404f71b93ea7de8d90e4/charts/vald/templates/rebalancer/jobconfigmap.yaml#L60